packages/batch-poster-monitor/index.ts

```diff
@@ -199,7 +199,7 @@ const getBatchPosterFromEventLogs = async (
   return tx.from
 }
 
-const getBatchPosterLowBalanceAlertMessage = async (
+const getBatchPosterAddress = async (
```

Comment:
I think I missed this last time (or maybe I asked, but I don't remember), sorry about that.

Why do we want to use the SDK to get the batch poster if we have the events already? It seems to me that it's faster to just get the batch poster from the events we have queried, and searching through the SDK doesn't add enough value to justify the extra calls.

Reply:
Can we add the condition to check from SDK if there are no logs?
Something like:
```
if (sequencerInboxLogs) {
  // Check from logs
}

if (!batchPoster) {
  // Check from SDK
}
```

In most cases we will have logs, and with the current implementation we are always making the extra log search from the SDK.

Reply:
Yeah actually that'd make more sense